asreview/review/base.py

```diff
@@ -180,6 +190,30 @@ def _prior_knowledge(self, logger):
                           np.zeros(len(self.prior_included)),
                           logger, method="initial")
 
+    @property
+    def settings(self):
+        if self.prior_included is not None:
+            n_prior_included = len(self.prior_included)
```

Comment:
Based on the GUI work, it might be useful to add prior_included to ASReviewSettings as well. Then we can move these statements to the ASReviewSettings class. What do you think?

Reply:
Yes, that seems like a good idea. Preferably the settings would be enough to rerun the simulation with the same settings.